rustic-cargo.el

```diff
@@ -455,17 +455,15 @@ As a byproduct, you can run any shell command in your project like `pwd'"
   "Run 'cargo run' for the current project.
 If running with prefix command `C-u', read whole command from minibuffer."
   (interactive "P")
-  (let* ((command (if arg
-                      (read-from-minibuffer "Cargo run command: " "cargo run -- ")
-                    (concat (rustic-cargo-bin) " run "
-                            (setq rustic-run-arguments
-                                  (read-from-minibuffer
-                                   "Run arguments: "
-                                   (if (rustic-cargo-run-get-relative-example-name)
-                                       (concat "--example "
-                                               (rustic-cargo-run-get-relative-example-name))
-                                     (car compile-history))
-                                   nil nil 'compile-history)) ))))
+  ;; TODO: benches detect if possible?
+  (let* ((example-name (rustic-cargo-run-get-relative-example-name))
+         (command (if arg
+                      (read-from-minibuffer "Cargo run command: " "cargo run ")
+                    (concat rustic-cargo-bin " run "
```

Comment:
`rustic-cargo-bin` should be a function call.

Reply:
Fixed